test/aqua/operators/test_abelian_grouper.py

```diff
@@ -53,23 +53,19 @@ def test_abelian_grouper2(self):
                 self.assertTrue(op_1.commutes(op_2))
 
     def test_abelian_grouper3(self):
```

Comment:
Instead of running 4 cominations internally you can split them using `ddt`, as
```python
@data((True, True), (True, False), (False, True), (False, False))
@unpack
def test_grouped_sum(self, fast, use_nx):
  ...
```

Reply:
Thank you for your information!